### DISCUSSION TITLE
[1132720]末世孤雄 Vol.0[本体]

### DISCUSSION THREAD

**Discussion by Alexikuns** [pjy612/SteamManifestCache#686] [Share]
### Steam 游戏ID

1132720

### 密钥列表

```hash
1132722;e7c4a6d945feb691be0b610d09ffcbc20057dd6423b2996f3769ec69340f4f4d
```

### 清单附件

[1132720.zip](https://github.com/user-attachments/files/15536698/1132720.zip)

### 备注

Email：1060312049@qq.com

**Comment by Alexikuns** [comment]
怎么加入内测啊

**Reply by pjy612** [reply]
无内测，可删帖
